core/src/components/segment-button/segment-button.ionic.scss

```diff
@@ -0,0 +1,107 @@
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+@use "./segment-button.common";
+
+// Ionic Segment Button
+// --------------------------------------------------
+
+:host {
+  /**
+   * @prop --color-disabled: Color of the disabled segment button
+   */
+  --background: #{globals.$ionic-color-base-white};
+  --background-checked: var(--background);
+  --color: #{globals.$ionic-color-neutral-1000};
+  --color-checked: #{globals.$ionic-color-primary-base};
+  --color-disabled: #{globals.$ionic-color-neutral-500};
+  --border-width: #{globals.$ionic-border-size-025};
+  --border-color: #{globals.$ionic-color-neutral-300};
+  --border-style: #{globals.$ionic-border-style-solid};
+  --indicator-box-shadow: none;
+  --indicator-color: var(--color-checked);
+  --indicator-height: #{globals.$ionic-border-size-025};
+  --indicator-transition: transform 250ms cubic-bezier(0.4, 0, 0.2, 1);
+  --indicator-transform: none;
+  --padding-top: #{globals.$ionic-space-200};
+  --padding-end: #{globals.$ionic-space-200};
+  --padding-bottom: #{globals.$ionic-space-200};
+  --padding-start: #{globals.$ionic-space-200};
+  --transition: color 0.15s linear 0s, opacity 0.15s linear 0s;
+
+  min-width: globals.$ionic-scale-1200;
+  min-height: globals.$ionic-scale-1200;
+}
+
+.button-native {
+  border-bottom: var(--border-width) var(--border-style) var(--border-color);
+
+  // Ensures the indicator moves correctly between segment buttons
+  z-index: 1;
+}
+
+.button-inner {
+  gap: globals.$ionic-space-100;
```

Comment:
This value seems to only apply when icon is top or bottom. If it's left or right, then it's `space-200`. Please confirm for which one it is.

Reply:
You are correct, it should be `space-200` when the layout is horizontal. Updated here: https://github.com/ionic-team/ionic-framework/pull/29897/commits/c1e7e3e49952afe2c7824eed67bd8ac0f931e76b